mythril/laser/ethereum/instructions.py

```diff
@@ -701,12 +701,15 @@ def codecopy_(self, global_state: GlobalState) -> List[GlobalState]:
                 )
             return [global_state]
 
-        bytecode = global_state.environment.code.bytecode
```

Comment:
I don't think this will ever happen (or does it?)

Reply:
Yes, you're right. It doesn't happen anymore because the prefix is removed earlier, there won't be a case where `global_state.environment.code.bytecode.startswith("0x") == True`